<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetaExplorer</class>
 <widget class="QWidget" name="MetaExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>500</height>
   </size>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="meta_explorer_layout">
     <item>
      <widget class="QGroupBox" name="meta_cover_group">
       <property name="toolTip">
        <string>Jaquette de l'album</string>
       </property>
       <property name="title">
        <string>Jaquette</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="meta_info_cover">
          <property name="minimumSize">
           <size>
            <width>370</width>
            <height>194</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Jaquette de l'album</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="textFormat">
           <enum>Qt::PlainText</enum>
          </property>
          <property name="pixmap">
           <pixmap resource="icons/icons.qrc">:/cover/img_missing.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="meta_info_group">
       <property name="toolTip">
        <string>Informations sur le morceau en cours</string>
       </property>
       <property name="title">
        <string>Informations</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <layout class="QVBoxLayout" name="meta_info_fields_layout">
          <item>
           <layout class="QHBoxLayout" name="meta_info_title_layout">
            <item>
             <widget class="QLabel" name="label_title">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Titre</string>
              </property>
              <property name="text">
               <string>Titre</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="meta_info_title">
              <property name="toolTip">
               <string>Titre</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="meta_info_album_layout">
            <item>
             <widget class="QLabel" name="label_album">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Album</string>
              </property>
              <property name="text">
               <string>Album</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="meta_info_album">
              <property name="toolTip">
               <string>Album</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="meta_info_author_layout">
            <item>
             <widget class="QLabel" name="label_author">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Artiste(s)</string>
              </property>
              <property name="text">
               <string>Artiste(s)</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="meta_info_author">
              <property name="toolTip">
               <string>Artiste(s)</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="meta_info_type_layout">
            <item>
             <widget class="QLabel" name="label_type">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Genre musicale</string>
              </property>
              <property name="text">
               <string>Genre</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="meta_info_type">
              <property name="toolTip">
               <string>Genre musicale</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="meta_info_year_layout">
            <item>
             <widget class="QLabel" name="label_year">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Année de production</string>
              </property>
              <property name="text">
               <string>Année</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="meta_info_year">
              <property name="toolTip">
               <string>Année de production</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="meta_info_comment_layout">
            <item>
             <widget class="QLabel" name="label_comment">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Commentaire(s)</string>
              </property>
              <property name="text">
               <string>Commentaire(s)</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="meta_info_comment">
              <property name="toolTip">
               <string>Commentaire(s)</string>
              </property>
              <property name="text">
               <string>-</string>
              </property>
              <property name="textFormat">
               <enum>Qt::PlainText</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="file_info_layout">
            <item>
             <spacer name="horizontal_spacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="meta_file_info">
              <property name="minimumSize">
               <size>
                <width>33</width>
                <height>29</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>33</width>
                <height>29</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Informations sur le fichier audio</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="icons/icons.qrc">
                <normaloff>:/sound/audio.png</normaloff>:/sound/audio.png</iconset>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
